### PR TITLE
Icon revamp

### PR DESCRIPTION
**Pull request by PikaDude**
This PR includes:
- Updated K.K. slider icon to feature his New Leaf render in a higher resolution
- Icons to now be stored in 32px, 64px and 128px formats for use on screens with different pixel densities and resolutions
- Alternate leaf status icons for when the weather is raining or snowing
- Increases the resolution of the clock and K.K. slider icons seen in desktop notifications

Name | Icon
--- | ---
Sunny | 
Raining | 
Snowing | 
Paused | 
K.K. Slider |

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -36,9 +39,5 @@
       "http://api.openweathermap.org/"
    ],
    "update_url": "https://clients2.google.com/service/update2/crx",
-   "version": "3.4",
-   "web_accessible_resources": [
-      "img/clock.png",
-      "sound/bells.ogg"
-   ]
```
Comment (jordigarcl) [inline manifest.json:43]: What was this`web_accessible_resources` array?

Comment (jordigarcl) [reply manifest.json:43]: If I understand https://developer.chrome.com/extensions/manifest/web_accessible_resources correctly it defined extension resources that could.be accessed from outside right?

Comment (PikaDude) [reply manifest.json:43]: No clue. Hasn't been updated in a while, and doesn't seem to actually effect anything so I removed it.

Comment (PikaDude) [reply manifest.json:43]: I believe so, however, this extension doesn't do anything like that so it isn't required.

Comment (jordigarcl) [reply manifest.json:43]: Yeah good call to remove this. Strange that this was here in the first place 😅

**Review by jordigarcl — APPROVED** [review]

**Pull request merged**